test/sql/copy/csv/csv_nullstr_list.test

```diff
@@ -0,0 +1,130 @@
+# name: test/sql/copy/csv/csv_nullstr_list.test
+# description: Test CSVs with multiple nullstring values
+# group: [csv]
+
+statement ok
+PRAGMA enable_verification
+
+# Test List
+query III
+FROM read_csv('data/csv/null/multiple_nulls.csv', auto_detect=false, delim=',', quote='"', escape='"', new_line='\n', skip=0, header=true, columns={'name': 'VARCHAR', 'age': 'VARCHAR', 'height': 'VARCHAR'}, nullstr = ['','none','null']);
```

Comment:
Can you test:

`nullstr = []`
`nullstr = ['a', NULL]`
`nullstr = NULL`
`nullstr = [42]`